OloEngine/src/OloEngine/Renderer/Debug/CommandPacketDebugger.cpp

```
@@ -1,834 +1,1198 @@
 {
+    CommandPacketDebugger& CommandPacketDebugger::GetInstance()
+    {
+        static CommandPacketDebugger instance;
+        return instance;
+    }
+
+    // ========================================================================
+    // Main entry point
+    // ========================================================================
+
     void CommandPacketDebugger::RenderDebugView(const CommandBucket* bucket, bool* open, const char* title)
```

Clang (14.0.6)

[warning] pointer parameter 'open' can be pointer to const

(readability-non-const-parameter)